temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -6,13 +6,15 @@ option go_package = "go.temporal.io/api/cloud/cloudservice/v1;cloudservice";
 
 import "temporal/api/cloud/operation/v1/message.proto";
 import "temporal/api/cloud/identity/v1/message.proto";
+import "temporal/api/cloud/namespace/v1/message.proto";
 
 message GetUsersRequest {
-    // The requested size of the page to retrieve
+    // The requested size of the page to retrieve - optional.
+    // Cannot exceed 1000. Defaults to 100. 
     int32 page_size = 1;
-    // The page token
+    // The page token if this is continuing from another response - optional.
     string page_token = 2;
-    // Optional field to filter users by email address
+    // Filter users by email address - optional
     string email = 3;
```

Comment:
Completely unrelated, feel free to punt this comment for another day.

Should we have allowed a list of emails? Then this can operate as a batch Get operation (as opposed to just single email). Ditto for IDs.

Reply:
:+1: At the very least this should clarify whether it's exact match or partial match